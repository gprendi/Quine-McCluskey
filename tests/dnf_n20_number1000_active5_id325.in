d 20
01000100001000000001
01000100001000000000
00000100000000000000
00000000000001000000
01000000000001000000
01000000000001000001
00000000000001000001
00000100001000000001
00000000001000000001
00000000001001000000
01000100001001000001
01000100000000000000
00000100001000000000
01000100000000000001
01000100000001000000
01000000001001000001
01000000001000000001
00000000000000000001
00000100000001000001
00000100001001000001
01000100000001000001
01000000001001000000
00000100000001000000
01000000000000000000
00000000000000000000
00000000001001000001
00000000001000000000
00000100001001000000
01000000001000000000
01000000000000000001
00000100000000000001
01000100001001000000
