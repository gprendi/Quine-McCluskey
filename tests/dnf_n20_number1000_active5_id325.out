p dnf 20 0
-1 -3 -4 -5 -7 -8 -9 -10 -12 -13 -15 -16 -17 -18 -19 0
